app/controllers/questions/authenticated_intake_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Questions
+  class AuthenticatedIntakeController < QuestionsController
```

Comment:
The setup in this controller looks hella similar to what's happening in the Portal::PortalController, which we're inheriting from in some of the sub-portal controllers. I'm wondering if we could consolidate. It seems like right now, that's not a particularly straightforward task (because this controller actually uses QuestionsController as its parent).  

I think this would require a significant refactor to do it in a way that would diminish complexity instead of increase it, but perhaps we can get around to it some day.

Reply:
I 100% agree! I considered putting the whole authenticated intake portion into the portal, but felt that in the short term that it made sense to distinguish between a protected portion of the intake and the portal. But I definitely feel like there is potential for good refactors in the future.